app/upgrades/v16/upgrades.go

```diff
@@ -28,9 +32,19 @@ func CreateUpgradeHandler(
 			}
 		}
 
+		if err := BurnUsageIncentivesPool(ctx, bankKeeper); err != nil {
+			logger.Error("failed to burn inflation pool", "error", err.Error())
+		}
+
+		if err := UpdateInflationParams(ctx, inflationKeeper); err != nil {
+			logger.Error("failed to update inflation params", "error", err.Error())
+		}
+
 		// recovery module is deprecated since it is renamed to "revenue" module
 		logger.Debug("deleting recovery module from version map...")
```

Comment:
is this comment accurate? I thought the recovery module has nothing to do with revenue